app/src/pages/inside/logsPage/logItemInfo/logItemInfoTabs/stackTrace/stackTrace.scss

```diff
@@ -0,0 +1,60 @@
+.stack-trace {
+  .stack-trace-content {
+    padding: 10px 40px 15px 10px;
+    background-color: $COLOR--light-red;
+  }
+}
+
+.stack-trace-message {
+  color: $COLOR--charcoal-grey;
+  line-height: 19px;
+  font-size: 12px;
+  font-family: Menlo, Monaco, Consolas, 'Courier New', monospace;
```

Comment:
Discuss using of this Font with UX Designer.

Reply:
Done